WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -270,6 +309,11 @@ private extension StoreStatsV4PeriodViewController {
         timeRangeBarView.backgroundColor = .systemColor(.secondarySystemGroupedBackground)
         visitorsStackView.backgroundColor = .systemColor(.secondarySystemGroupedBackground)
 
+        // Visitor empty view - insert it at the second-to-last index
+        let emptyViewIndex = max(0, visitorsStackView.arrangedSubviews.count - 2)
```

Comment:
why does it have to be inserted at the second-to-last index? some comments would also help here 🙂

Reply:
Since we have the footer view at the bottom of the stack view to push content to the center, the empty view needs to be inserted at the second-to-last index 😄. I have updated the comment in f76335f8cd4c96b4b08bf95b3e0af5a6f17d5aff.